### PR TITLE
ab preview: start time and duration bug

### PR DESCRIPTION
**Pull request by annagav**
Start time and Duration were not appearing in the setup experiment table.

### CODE REVIEW

File: html_app/model/Treatment.js [hunk 1]
```diff
@@ -35,85 +35,91 @@ scb.Treatment = function scb_Treatment(data, context, parent) {
...
 	// should be getter only
-	scb.Utils.initialize_accessor_field(self, data, 'schedule_value', "0", null, context);
+	scb.Utils.initialize_accessor_field(self, data, 'start_time_value', "0", null, context);
 	// should be getter only
 	scb.Utils.initialize_accessor_field(self, data, 'duration_value', "0", null, context);
+    if(data.start_time_value !== "0"){
+        scb.Utils.initialize_field(data, 'start_time', self.schedule_value);
+        Object.defineProperty(self, 'start_time', {
+            get : function() {
+                var time = parseFloat(data.schedule_value);
```
Comment (noisecapella) [inline html_app/model/Treatment.js:45]: Can this be renamed `seconds`?

Comment (annagav) [inline html_app/model/Treatment.js:45]: This is the old code that is not used by current assignments. I think we will be deleting it soon.

**Comment by noisecapella** [comment]
By the way I see a word `collection_ab` in the Collection Time column but that's out of the scope of this PR

File: html_app/model/Treatment.js (hunk 1 above)
Comment (noisecapella) [inline html_app/model/Treatment.js:45]: That's fine

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**